--- 01_dtbdump_Qualcomm_Technologies,_Inc._ARA-ER_SDM_636_PM660_+_PM660L_QRD.dts	2025-02-15 00:58:24.598591461 +0000
+++ 01_dtbdump_Qualcomm_Technologies,_Inc._ARA-ER_SDM_636_PM660_+_PM660L_QRD_after.dts	2025-02-15 01:58:50.466155158 +0000
@@ -14802,17 +14802,27 @@
 
 				vendor {
 					compatible = "android,vendor";
-					dev = "/dev/block/platform/soc/c0c4000.sdhci/by-name/vendor";
+					dev = "/dev/block/platform/soc/c0c4000.sdhci/../c084000.sdhci/by-name/microsd_vendor";
 					type = "ext4";
-					mnt_flags = "ro,barrier=1,discard";
+					mnt_flags = "rw,barrier=1,discard";
 					fsmgr_flags = "wait";
 					status = "ok";
 				};
 
 				system {
 					compatible = "android,system";
-					dev = "/dev/block/platform/soc/c0c4000.sdhci/by-name/system";
+					dev = "/dev/block/platform/soc/c0c4000.sdhci/../c084000.sdhci/by-name/microsd_system";
+					type = "ext4";
+					mnt_flags = "rw,barrier=1,discard";
+					fsmgr_flags = "wait";
+					status = "ok";
+				};
+
+				cache {
+					compatible = "android,cache";
+					dev = "/dev/block/platform/soc/c0c4000.sdhci/by-name/cache";
 					type = "ext4";
+					mnt_point = "/vendor/internalcachesothatwecanbootfromsdcard";
 					mnt_flags = "ro,barrier=1,discard";
 					fsmgr_flags = "wait";
 					status = "ok";
